LogMonitor/src/LogMonitor/EventMonitor.h

```diff
@@ -25,6 +26,14 @@ class EventMonitor final
     const std::vector<EventLogChannel> m_eventChannels;
     bool m_eventFormatMultiLine;
     bool m_startAtOldestRecord;
+    std::wstring m_logFormat;
+
+    std::wstring source;

```

Comment:
here, thinking this could be encapsulated in a struct; as in my other comment. Can do a refactor after the merge.

Reply:
nit: we can also condense the declaration into one line: std:wstring eventTime, eventChannel, eventLevel, eventMessage, source;

Reply:
@iankingori  -- actually when they are many like that ends-up hampering readability...

Reply:
The change will be addressed in a new PR after merge as agreed.